daemon/src/projection.rs

```diff
@@ -789,15 +786,20 @@ impl Tx {
         let _ = self.quote.send(quote.map(|q| q.into()));
     }
 
-    fn send_offer_update(&self, offer: MakerOffers) {
-        self.offer.send(offer).unwrap_or_default();
+    fn send_offer_update(&self, offers: MakerOffers) -> Result<()> {
+        tracing::info!(?offers, "Sending maker offers to feed");
```

Comment:
I think I was debugging and this slipped through. I will make sure I remove any such log statements.

Reply:
I think it's actually good to introduce a log statement that logs the offers - I don't think we have one since the new protocol was introduced. So feel free to keep one somewhere :)